src/RealSensePlugin.cpp

```diff
@@ -132,21 +147,21 @@ void RealSensePlugin::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf)
   this->world = this->rsModel->GetWorld();
 
   // Sensors Manager
-  sensors::SensorManager * smanager = sensors::SensorManager::Instance();
-
+  smanager = sensors::SensorManager::Instance();
   // Get Cameras Renderers
+  sensors_arr = smanager->GetSensors();
+
   this->depthCam = std::dynamic_pointer_cast<sensors::DepthCameraSensor>(
-    smanager->GetSensor(prefix + DEPTH_CAMERA_NAME))
+    FindSensor(prefix + DEPTH_CAMERA_NAME, this->rsModel->GetName()))
```

Comment:
One of the solution was to add a prefix of amr_id to camera name but it wasn't possible as our topics would look like /agv2/realsense_left2/smth but in forklift_params.yaml we need specify topics to use for stvl and those would be different for different AMRs. But if you know a solution to that please comment